add documentation for MD flowsheet (#1444)
watertap-org/watertap: docs/technical_reference/flowsheets/membrane_distillation.rst

    .. math::

        LCOW_{Q} = \frac{f_{crf} \cdot C_{cap,tot} + C_{op,tot}}{f_{util} \cdot Q}

Degrees of Freedom
------------------

In addition to the overall water recovery, the following variables need to be specified based on the default settings:
   * Feed water conditions (flow, temperature, pressure, component concentrations)
   * Pump efficiency
   * Membrane permeability coefficient, thickness, and thermal conductivity
   * MD Hot and cold channel spacer porosity and height
   * Heat exchanger overall heat transfer coefficient

Flowsheet Specifications
------------------------

.. csv-table::
   :header: "Description", "Units", "Value"

   "**MD Module**"
   "Membrane permeability coefficient", ":math:`\text{m/Pa/s}`", "1e-10"
   "Membrane thickness", ":math:`\text{m}`", "1e-4"
   "Membrane thermal conductivity", ":math:`\text{W/m/K}`", "0.2"
   "MD hot and cold channels height", ":math:`\text{m}`", "0.0019"
   "MD hot and cold channels spacer porosity", ":math:`\text{dimensionless}`", "0.77"
   "**Heat Exchanger**"
   "Overall heat transfer coefficient", ":math:`\text{W/m}^2\text{K}`", "2000"
   "**Pumps**"

>    "Efficiency", ":math:`\text{dimensionless}`", "0.8"

Reviewer: The "Units" and "Value" columns here should be swapped to match other flowsheet documentation, and it also just looks nicer to have the units after the value

Degrees of Freedom
------------------

In addition to the overall water recovery, the following variables need to be specified based on the default settings:
   * Feed water conditions (flow, temperature, pressure, component concentrations)
   * Pump efficiency
   * Membrane permeability coefficient, thickness, and thermal conductivity
   * MD Hot and cold channel spacer porosity and height
   * Heat exchanger overall heat transfer coefficient

Flowsheet Specifications
------------------------

.. csv-table::
   :header: "Description", "Units", "Value"

   "**MD Module**"
   "Membrane permeability coefficient", "1e-10", "m/Pa/s"
   "Membrane thickness", "1e-4", "m"
   "Membrane thermal conductivity", "0.2", "W/m/K"
   "MD hot and cold channels height", "0.0019", "m"
   "MD hot and cold channels spacer porosity", "0.77", "dimensionless"
   "**Heat Exchanger**"
   "Overall heat transfer coefficient", "2000", "W/m²K"
   "**Pumps**"
   "Efficiency", "0.8", "dimensionless"
   "**Heater**"
   "Maximum outlet temperature", "90", "°C"
   "**Cooler**"
   "Minimum outlet temperature", "10", "°C"

Additional Variables
--------------------

.. csv-table::

>    :header: "Description", "Symbol", "Units"
> 
>    "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recyle_ratio``, ":math:`\text{dimensionless}`"

Reviewer: ```suggestion
   :header: "Description", "Variable Name", "Units"

   "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recycle_ratio``, ":math:`\text{dimensionless}`"
```

or

```suggestion
   :header: "Description", "Symbol", "Units"

   "Recycle ratio (Ratio of recycle stream to the make up feed)", ``R``, ":math:`\text{dimensionless}`"
```

Reviewer: I like the first one more, but if you choose the second one, feel free to change the symbol from just `R`

Author: changed "symbol" to "variable name" because "symbol" is unnecessary in this context, and "variable name" is more useful.

.. figure:: ../../_static/flowsheets/MD.png
    :width: 600
    :align: center

    Figure 1. MD flow diagram

Documentation for each of the unit models can be found here:
   * `MD <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/membrane_distillation_0D.html>`_
   * `Heat Exchanger <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html>`_
   * `Heater and chiller <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/heater.html>`_
   * `Mixer <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/mixer.html>`_
   * `Separator <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/separator.html>`_

Documentation for the property model can be found here:
   * `Seawater Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_
   * `Water Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_

> This flowsheet aims to solve an optimization problem that minimizes the levelized cost of water (LCOW) with a specified overall water recovery.

Reviewer 2: ```suggestion
This flowsheet aims to solve an optimization problem that minimizes the LCOW with a specified overall water recovery.
```

LCOW acronym defined in implementation section.

LCOW can be represented by the following equation where :math:`Q` represents product volumetric flow, :math:`f_{crf}` represents the capital recovery factor, :math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and :math:`f_{util}` represents the utilization factor:

    .. math::

        LCOW_{Q} = \frac{f_{crf} \cdot C_{cap,tot} + C_{op,tot}}{f_{util} \cdot Q}

Degrees of Freedom
------------------

In addition to the overall water recovery, the following variables need to be specified based on the default settings:
   * Feed water conditions (flow, temperature, pressure, component concentrations)
   * Pump efficiency
   * Membrane permeability coefficient, thickness, and thermal conductivity

>    * MD Hot and cold channel spacer porosity and height

Reviewer 2: ```suggestion
   * MD hot and cold channel spacer porosity and height
```

   * Heat exchanger overall heat transfer coefficient

Flowsheet Specifications
------------------------

.. csv-table::

>    :header: "Description", "Units", "Value"

Reviewer 2: Units and Value column headers need to be switched.

>    :header: "Description", "Units", "Value"

Reviewer 2: ```suggestion
   :header: "Description", "Value", "Units"
```

   "**MD Module**"

>    "Membrane permeability coefficient", "1e-10", "m/Pa/s"

Reviewer 2: ```suggestion
   "Membrane permeability coefficient", "1e-10", ":math:`\text{m/Pa/s}`"
```

>    "Membrane thickness", "1e-4", "m"

Reviewer 2: ```suggestion
   "Membrane thickness", "1e-4", ":math:`\text{m}`"
```

>    "Membrane thermal conductivity", "0.2", "W/m/K"

Reviewer 2: ```suggestion
   "Membrane thermal conductivity", "0.2", ":math:`\text{W/m/K}`"
```

>    "MD hot and cold channels height", "0.0019", "m"

Reviewer 2: ```suggestion
   "MD hot and cold channels height", "0.0019", ":math:`\text{m}`"
```

>    "MD hot and cold channels spacer porosity", "0.77", "dimensionless"

Reviewer 2: ```suggestion
   "MD hot and cold channels spacer porosity", "0.77", ":math:`\text{dimensionless}`"
```

   "**Heat Exchanger**"

>    "Overall heat transfer coefficient", "2000", "W/m²K"

Reviewer 2: ```suggestion
   "Overall heat transfer coefficient", "2000", ":math:`\text{W/m}^{2}\text{/K}`"
```

   "**Pumps**"

>    "Efficiency", "0.8", "dimensionless"

Reviewer 2: ```suggestion
   "Efficiency", "0.8", ":math:`\text{dimensionless}`"
```

   "**Heater**"

>    "Maximum outlet temperature", "90", "°C"

Reviewer 2: ```suggestion
   "Maximum outlet temperature", "363", ":math:`\text{K}`"
```

   "**Cooler**"

>    "Minimum outlet temperature", "10", "°C"

Reviewer 2: ```suggestion
   "Minimum outlet temperature", "283", ":math:`\text{K}`"
```

Author: We can keep the temperature units in °C . This is consistent with the flowsheet results display where temperatures are also reported in °C."

Reviewer 2: That is ok with me!

Additional Variables
--------------------

.. csv-table::
   :header: "Description", "Variable Name", "Units"

>    "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recyle_ratio``, ":math:`\text{dimensionless}`"

Reviewer: ```suggestion
   "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recycle_ratio``, ":math:`\text{dimensionless}`"
```

>    "Recycle ratio (Ratio of recycle stream to the make up feed)", ``Recycle_ratio``, ":math:`\text{dimensionless}`"

Reviewer 2: ```suggestion
   "Recycle ratio (Ratio of recycle stream to the make up feed)", ``recycle_ratio``, ":math:`\text{dimensionless}`"
```

Additional Constraints
----------------------

> There is an extra inequality constraint to ensure the Liquid Entry Pressure (LEP) to prevent membrane wetting:

Reviewer 2: slightly confusing sentence... 

should it be something like "...to ensure the Liquid Entry Pressure (LEP) is sufficiently low to prevent membrane wetting"?

Author: Good catch. Thanks. modified to " There is an extra inequality constraint to ensure the hydraulic pressure inside the MD module is lower than the Liquid Entry Pressure (LEP) to prevent membrane wetting"
